spinnaker_graph_front_end/examples/hello_world_untimed/hello_world.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) 2017-2019 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+Hello World program on SpiNNaker
+
+Each core stores into its region in SDRAM the string:
+"Hello World from $chip.x, $chip.y, $core"
+
+We then fetch the written data and print it on the python console.
+"""
+
+import logging
+import os
+import spinnaker_graph_front_end as front_end
+from spinnaker_graph_front_end.examples.hello_world_untimed\
+    .hello_world_vertex import HelloWorldVertex
+
+logger = logging.getLogger(__name__)
+
+front_end.setup(
+    n_chips_required=1, model_binary_folder=os.path.dirname(__file__))
+
+# Put HelloWorldVertex onto 16 cores
+total_number_of_cores = 16
+prints_per_run = 10
+end_scale = 2
+for x in range(total_number_of_cores):
+    front_end.add_machine_vertex_instance(
+        HelloWorldVertex(x * end_scale,
+                         label="Hello World {}".format(x)))
+
+runs = ((total_number_of_cores * end_scale) // prints_per_run) + 1
+for _ in range(runs):
+    front_end.run_until_complete(prints_per_run)
```

Comment:
ok. so what happens if i say run for many more times than n_repeats? say 1 billion for prints per run? as that well different to x * end_scale. i think youd get a mismatch somewhere

Reply:
I have removed this as I can see why you might have been confused.  It would have worked though...